Office 365/o365/_meta/fields.yml

```
@@ -343,3 +343,13 @@ email.attachments:
   name: email.attachments
   description: "A list of objects describing the attachment files sent along with an email message"
   type: array
+
+email.subjects:
+  name: email.subjects
+  description: "A list of email subjects"
+  type: array
+
+email.from.domain:
+  name: email.from.domain
+  description: "The domain of the sender. Might be an array"
+  type: keyword
```

Custom fields must not look similar to ECS one. They must be prefixed with the name of the vendor and the name of the format (see [Taxonomy](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/taxonomy.md#custom-taxomony))
```suggestion
office365.investigation.email.subjects:
  name: office365.investigation.email.subjects
  description: "A list of email subjects"
  type: array

office365.investigation.email.sender.domains:
  name: office365.investigation.email.sender.domains
  description: "The domain of the sender. Might be an array"
  type: keyword
```

(Do not account of the `email.*` fields declared in this taxonomy; they date from a time the ECS taxonomy was not up-to-date).